src/app/shared/deactivate-device/deactivate-device.component.ts

```diff
@@ -0,0 +1,78 @@
+import { Component, Inject, OnInit } from '@angular/core'
+import { FormBuilder, FormGroup, Validators } from '@angular/forms'
+import { MAT_DIALOG_DATA } from '@angular/material/dialog'
+import { MatTabChangeEvent } from '@angular/material/tabs'
+import { environment } from 'src/environments/environment'
+import { VaultService } from './vault.service'
+
+@Component({
+  selector: 'app-deactivate-device',
+  templateUrl: './deactivate-device.component.html',
+  styleUrls: ['./deactivate-device.component.scss']
+})
+export class DeactivateDeviceComponent implements OnInit {
+  vaultForm: FormGroup
+  hasPasswordretrieved: boolean = false
+  rpcLinux: string = 'sudo ./rpc '
+  rpcDocker: string = 'sudo docker run --device=/dev/mei0 rpc:latest '
+  rpcWindows: string = 'rpc.exe '
+  deactivationUrl: string = ''
+  serverUrl: string = `-u ${environment.rpsServer.replace('http', 'ws').replace('/rps', '')}/activate -c`
```

Comment:
won't work if not using kong see suggestion

Reply:
Updated